frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/GitFetchSupportImpl.java

```diff
@@ -89,8 +89,7 @@ public FetchResultImpl fetchDefaultRemote(Collection<GitRepository> repositories
             return false;
           }
 
-        }).map(repoAndRemote -> new RemoteRefCoordinates(repoAndRemote._1(), repoAndRemote._2(), /* refspec */ null))
-        .collect(List.collector());
```

Comment:
~Good point! Maybe some regexp for that?~
![image](https://user-images.githubusercontent.com/19799111/84787620-819f4f00-afee-11ea-91b2-06e423e5e45c.png)

okay, sometimes it actually is needed... but can be managed like this from:
![image](https://user-images.githubusercontent.com/19799111/84788743-c2e42e80-afef-11ea-94b5-75f937cfb111.png)
to:
![image](https://user-images.githubusercontent.com/19799111/84788686-b364e580-afef-11ea-9bfb-ae6128de539e.png)

Reply:
:trollface:

Reply:
Done indeed (via `FenumChecker`) on PR #329 :bangbang:

Reply:
See https://github.com/VirtusLab/git-machete-intellij-plugin/pull/329/commits/13058bb43398213101b1c7beb4197e7afb822789

Reply:
cool! :rocket: